src/Okta.Sdk.UnitTests/OktaClientShould.cs

```diff
@@ -9,13 +9,64 @@
 using System.Threading;
 using System.Threading.Tasks;
 using FluentAssertions;
+using NSubstitute;
+using Okta.Sdk.Internal;
 using Okta.Sdk.UnitTests.Internal;
 using Xunit;
 
 namespace Okta.Sdk.UnitTests
 {
     public class OktaClientShould
     {
+        [Fact]
+        public void GetClientWithContentType()
+        {
+            var client = new TestableOktaClient(Substitute.For<IRequestExecutor>());
+            client.RequestContext.ContentType.Should().BeNullOrEmpty();
+            var requestContext = new RequestContext{ ContentType = "foo" };
+            var scopedClient = client.CreatedScoped(requestContext);
+            ((OktaClient)scopedClient).RequestContext.ContentType.Should().Be("foo");
+        }
+
+        [Fact]
+        public void GetClientWithContentTransferEncoding()
+        {
+            var client = new TestableOktaClient(Substitute.For<IRequestExecutor>());
+            client.RequestContext.ContentTransferEncoding.Should().BeNullOrEmpty();
+            var scopedClient = client.CreatedScoped(new RequestContext { ContentTransferEncoding = "foo" });
+            ((OktaClient)scopedClient).RequestContext.ContentTransferEncoding.Should().Be("foo");
+        }
+
+        [Fact]
+        public async Task OverrideRequestContentType()
+        {
+            var executor = Substitute.For<IRequestExecutor>();
+            executor
+                .PostAsync(Arg.Any<HttpRequest>(), Arg.Any<CancellationToken>())
+                .Returns(Task.FromResult(new HttpResponse<string>() { StatusCode = 200 }));
+
+            var client = new TestableOktaClient(executor);
+            await client.CreatedScoped(new RequestContext { ContentType = "foo" }).PostAsync(new HttpRequest { ContentType = "bar" });
```

Comment:
Let's mention in the README that `RequestContext.ContentType` > `HttpRequest.ContentType`

Reply:
Updated readme: https://github.com/okta/okta-sdk-dotnet/pull/392/commits/258e61c8b7fa765a206cea8728d9e2fa06856b9f